README.md

```diff
@@ -1217,13 +1217,13 @@ While this guide explains the *what*, *why* and *how*, I find it helpful to see
       var directive = {
           restrict: 'EA',
           templateUrl: 'app/feature/example.directive.html',
-          scope: {
+		  scope: {},
```

Comment:
`scope: true` means creating a child scope, while `scope: {}` creates an isolated scope. An isolated scope should be preffered